src/components/Popup.js

```diff
@@ -17,6 +18,14 @@ export default class Popup {
     this.#removeHandleEscClose();
   }
 
+  changeButtonText (isLoading) {
+    if (isLoading) {
+      this.submitButton.textContent = "Saving...";
+    } else {
+      this.submitButton.textContent = "Save";
+    }
+  }
```

Comment:
Please, move `changeButtonText` into `PopupWithForm` and `PopupDeleteCard` because the image modal doesn't have a submit button, so you can't keep this method in `Popup`